src/genophenocorr/analysis/_config.py

```diff
@@ -108,16 +137,20 @@ def pval_correction(self, pval_correction: typing.Optional[str]):
         """
         Set `pval_correction` option.
         """
-        # TODO - check admissible values
-        self._pval_correction = pval_correction
+        if pval_correction in P_VAL_OPTIONS:
+            self._pval_correction = pval_correction
+        else:
+            self._logger.warning('Ignoring invalid `pval_correction` value %s. Not doing p-value correction.', pval_correction)
         return self
 
     def min_perc_patients_w_hpo(self, min_perc_patients_w_hpo: float):
         """
         Set `min_perc_patients_w_hpo` option.
         """
-        # TODO - check float in range [0,1)
-        self._min_perc_patients_w_hpo = min_perc_patients_w_hpo
+        if min_perc_patients_w_hpo > 1 or min_perc_patients_w_hpo <= 0:
+            self._logger.warning("min_perc_patients_w_hpo must be greater than 0 and at most 1. Using default of 0.1")
```

Comment:
Showing logging messages such as this can be helpful but also deceiving if not done right. 

If we update the default value, in this case on line 120, we will almost *surely* forget to update the logging message. I think you should parameterize the logging message to print the current value of `self._min_perc_patients_w_hpo`. Then it will be futureproof.

Reply:
Fixed this!